### DISCUSSION TITLE
How to get a CODE?

### DISCUSSION THREAD

**Discussion by lrpauley13** [EtienneAb3d/karaok-AI#9] [General]
I have your software running on my PC, but no matter what I try I cannot get a code.  I select a song, then click on KAI. It asks for an email and the Command Prompt in the background says a code is sent to my email, but nothing happens.  I never get a code.

Is this a code to use Whisper?

**Comment by EtienneAb3d** [comment]
@lrpauley13 
Did you have a look at your spam folders?
I will send you your last key to you with an other of my email address that could possibly pass your spam filters.
It's not a code to use Whisper, but a code to use my own transcription server.

**Comment by lrpauley13** [comment]
I got it working.


Sent from Yahoo Mail for iPhone


On Sunday, December 1, 2024, 5:24 AM, Etienne Monneret ***@***.***> wrote:




@lrpauley13
Did you have a look at your spam folders?
I will send you your last key to you with an other of my email address that could possibly pass your spam filters.
It's not a code to use Whisper, but a code to use my own transcription server.

**Comment by lrpauley13** [comment]
Is there a way to change the color of the current lyric and the previous/next lyric?
    On Sunday, December 1, 2024 at 05:24:24 AM EST, Etienne Monneret ***@***.***> wrote:  
 
 


@lrpauley13
Did you have a look at your spam folders?
I will send you your last key to you with an other of my email address that could possibly pass your spam filters.
It's not a code to use Whisper, but a code to use my own transcription server.

**Reply by EtienneAb3d** [reply]
@lrpauley13 
This was planed, and the code is partially written, but not already fully operational.

**Comment by lrpauley13** [comment]
Thank you.


Sent from Yahoo Mail for iPhone


On Sunday, December 1, 2024, 10:38 AM, Etienne Monneret ***@***.***> wrote:




@lrpauley13
This was planed, and the code is partially written, but not already fully operational.
